docs/README.md

```diff
@@ -0,0 +1,51 @@
+# Eventbrite JavaScript SDK Documentation
+
+This SDK interface closely mirors the [Eventbrite v3 REST API](https://www.eventbrite.com/developer/v3/) endpoints that it wraps. The SDK provides many conveniences for making requests and processing responses to make it easier to use in the JavaScript environment.
+
+## ToC
+
+*   [Including the package](#including-the-package)
+*   [Configuring a SDK object](#configuring-a-sdk-object)
+*   [`request()`](./request)
+
+## Including the package
+
+First include the `eventbrite` package (depending on your module environment):
+
+### Webpack / Rollup / etc ([ECMAScript modules](https://unpkg.com/eventbrite/lib/esm/)):
+
+```js
+import eventbrite from 'eventbrite';
+```
+
+### Node / legacy dependency systems ([CommonJS](https://unpkg.com/eventbrite/lib/cjs/) / [Universal Module Definition](https://unpkg.com/eventbrite/lib/umd/)):
+
+```js
+const eventbrite = require('eventbrite');
+```
+
+### `<script>` [distribution bundle](https://unpkg.com/eventbrite/dist/) include:
+
+```
+<script src="https://unpkg.com/eventbrite/dist/eventbrite.min.js"></script>
+```
+
+NOTE: `window.Eventbrite` will be a reference to the package.
+
+## Configuring a SDK object
+
+In order to make requests, we need to configure the SDK object.
```

Comment:
nit: might change the 'we' to 'you' since you use 'you' in later sentences

Reply:
good catch